core/vm/contract.go

```diff
@@ -30,7 +30,7 @@ type ContractRef interface {
 // AccountRef implements ContractRef.
 //
 // Account references are used during EVM initialisation and
-// it's primary use is to fetch addresses. Removing this object
+// its primary use is to fetch addresses. Removing this object
```

Comment:
`it's ` seems correct to me -- it's a possessive pronoun

Reply:
OK, I will modify it back to "it's". My English is not native.

Reply:
Yeah, me neither, I had to look it up :)